modules/virt-about-reclaiming-persistent-volumes.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// virt/virtual_machines/virtual_disks/virt-reclaiming-persistent-volumes.adoc
+
+[id="virt-about-reclaiming-persistent-volumes_{context}"]
+
+= About reclaiming persistent volumes
+
+When you reclaim a volume, you unbind the PV from a persistent volume claim (PVC)
```

Comment:
s/volume/persistent volume (PV)

Reply:
When you reclaim a PV, you unbind the PV from a persistent volume claim (PVC)
and delete the data in the network share. Finally, you bind the PV to a new PVC to
prepare the volume for reuse.